fmpz.h

```diff
@@ -770,45 +775,8 @@ FLINT_DLL void _fmpz_CRT_ui_precomp(fmpz_t out, const fmpz_t r1, const fmpz_t m1
 FLINT_DLL void fmpz_CRT_ui(fmpz_t out, const fmpz_t r1, const fmpz_t m1,
     ulong r2, ulong m2, int sign);
 
-#define FLINT_FMPZ_LOG_MULTI_MOD_CUTOFF 2
```

Comment:
Shouldn't we deprecate this stuff over a number of versions instead of just removing it wholesale before people have been notified?